### DISCUSSION TITLE
Multiple logos when adding models

### DISCUSSION THREAD

**Discussion by pmbnevoa** [open-webui/open-webui#3504] [General]
# Bug Report

## Description

**Bug Summary:**
When a new chat is opened, a new logo (favicon) is displayed everytime a model is added.

**Steps to Reproduce:**

1. Log in to Open WebUI.
2. Select "New Chat" on the top left corner.
3. Click on the "+" button next to "Select a model" multiple times.

**Expected Behavior:**
A new line should be displayed to choose a model for each click on "+". No other changes should happen.

**Actual Behavior:**
A new line is displayed to choose a model. A new logo (favicon) is displayed above the welcome message for each time there is a click on the "+" button next to "Select a model".

## Environment

- **Open WebUI Version:** 0.3.6

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Screenshots (if applicable):**
![image](https://github.com/open-webui/open-webui/assets/63667819/ec63f608-10dd-43e4-91b8-6e53f7e5e63a)

## Installation Method

Docker.

**Comment by justinh-rahb** [comment]
Working as intended.

**Reply by pmbnevoa** [reply]
So it is supposed to see the logo above the welcome message multiple times?

**Reply by justinh-rahb** [reply]
Yes, to indicate that you are having a multiple-models conversation. Admittedly, this is a subjective style choice, but not an issue.

**Comment by pmbnevoa** [comment]
Found a workaround so that the logos are not duplicated.

In `\src\lib\components\chat\Messages\Placeholder.svelte` add `{#if modelIdx === 0} {/if}`:

```
{#if modelIdx === 0}
	<img
		crossorigin="anonymous"
		src={model?.info?.meta?.profile_image_url ??
			($i18n.language === 'dg-DG'
				? `/doge.png`
				: `${WEBUI_BASE_URL}/static/favicon.png`)}
		class=" size-[2.7rem] rounded-full border-[1px] border-gray-200 dark:border-none"
		alt="logo"
		draggable="false"
	/>
{/if}
```
